drive/files.py

```diff
@@ -257,34 +265,37 @@ def get_bytes(self, mime_type=None):
         fh.seek(0)
         return fh
 
-    def json(self):
+    def json(self) -> Any:
         """
         Convenient method to download a file as JSON.
         :return:
         """
         return json.load(self.get_bytes())
 
-    def jsons(self):
+    def jsons(self) -> Generator[Any]:
         """
         Convenient method to download a file as a newline-delimited list of JSON objects. This returns a generator.
         :return:
         """
-        line = True
+        line = b'1'
```

Comment:
This keeps the typing simple. Otherwise this variable needs to be annotated as `bool | bytes`.